### PR TITLE
Use nvm on teamcity

### PR DESCRIPTION
**Pull request by sndrs**
## What does this change?

uses NVM on teamcity

## What is the value of this and can you measure success?

currently only uses 6 but if this works will update to 8

## Does this affect other platforms - Amp, Apps, etc?

¯\_(ツ)_/¯

### CODE REVIEW

**Comment by sndrs** [comment]
@TBonnin currently failing with `nvm not found` error?

**Comment by sndrs** [comment]
guessing we need to `source` something?

**Review by SiAdcock — APPROVED** [review]
Nice let's do this (or more cautiously, let's deploy and carefully monitor this for the rest of the day and hopefully we won't break anything!)

File: dev/teamcity/dist-assets-tc [hunk 1]
```diff
@@ -4,6 +4,9 @@ set -o xtrace
 set -o nounset
 set -o errexit
 
+. ~/.nvm/nvm.sh
+nvm install
```
Comment (TBonnin) [inline dev/teamcity/dist-assets-tc]: what is this `nvm install` for?

Comment (sndrs) [reply dev/teamcity/dist-assets-tc]: will install the version specified in `nvmrc` then `use` it. if it's already installed just moves straight to `use`

**Pull request merged**